src/query/graphite/ts/series_reducer.go

```diff
@@ -56,9 +61,13 @@ func (sa SeriesReducerApproach) Reducer() SeriesReducer {
 
 var seriesReducers = map[SeriesReducerApproach]SeriesReducer{
 	SeriesReducerAvg:    func(b *Series) float64 { return b.SafeAvg() },
+	SeriesReducerAverage:    func(b *Series) float64 { return b.SafeAvg() },
+	SeriesReducerTotal:    func(b *Series) float64 { return b.SafeSum() },
 	SeriesReducerSum:    func(b *Series) float64 { return b.SafeSum() },
 	SeriesReducerMin:    func(b *Series) float64 { return b.SafeMin() },
 	SeriesReducerMax:    func(b *Series) float64 { return b.SafeMax() },
 	SeriesReducerStdDev: func(b *Series) float64 { return b.SafeStdDev() },
 	SeriesReducerLast:   func(b *Series) float64 { return b.SafeLastValue() },
+	SeriesReducerCurrent:   func(b *Series) float64 { return b.SafeLastValue() },
```

Comment:
Can you run gofmt on this file? Seems to be not aligned (which gofmt fixes ":" alignment).

Reply:
Done!